net.c

```diff
@@ -2540,13 +2530,11 @@ void net_update_state()
     if(strcmp(sl->key, "lo") && !strcmp(sl->value, "up")) slist_append_str(&config.ifcfg.if_up, sl->key);
   }
 
-  if(config.debug) {
-    fprintf(stderr, "net_update_state: ");
-    for(sl = config.ifcfg.if_state; sl; sl = sl->next) {
-      fprintf(stderr, "%s: %s%s", sl->key, sl->value, sl->next ? ", " : "");
-    }
-    fprintf(stderr, "\n");
+  log_debug("net_update_state: ");
+  for(sl = config.ifcfg.if_state; sl; sl = sl->next) {
```

Comment:
well, not exactly same code as it iterate even when debug is disabled, but I do not expect performance change.